.node-version

```diff
@@ -0,0 +1 @@
+12.13.0
```

Comment:
This is pretty old. Latest version is `12.18.0`.

Reply:
Ah, I just went with what I had installed, assuming it was largely sensible

Reply:
:)